.circleci/config.yml

```diff
@@ -35,6 +35,10 @@ jobs:
           name: Validate pytest-benchmark invocation of training suite
           command: ./scripts/run_bench_and_upload.sh
           no_output_timeout: "60m"
+      - run:
+          name: Sanity check for lazy_bench.py training suite
+          command: python lazy_bench.py --just_run_once  --run_in_subprocess alexnet --test=eval
```

Comment:
is this mainly to see if lazy bench throws?  e.g. still need another test in order to ensure correctly 'compiling' via looking at 
metrics

Reply:
this is literally to make sure lazy_bench.py runs to a completion. this isn't to test that LT works correctly or even runs)